.github/workflows/android.yml

```
@@ -0,0 +1,42 @@
+name: Build APK
+
+env:
+  main_project_module: app
+
+on:
+
+  push:
+    branches:
+      - 'release/**'
```

```suggestion
      - '*'
      - '!main'
```
this should trigger the workflow on all branches except main, which'll trigger another one